app.js

```diff
@@ -9,10 +9,13 @@ const session = require( 'express-session' )
 const models = require( './models/index' )
 const passport = require( './auth/passport' )
 const protectRoute = require( './auth/protectRoute' )
+const verifyJwt = require('./auth/verify-jwt')
 
 const routes = require( './routes/index' )
 const accounts = require( './routes/accounts' )
 const items = require( './routes/items' )
+const apiRoutes = require( './routes/api/accounts' )
```

Comment:
Multiple includes for api feels like it will become unwieldy - is there a better way to organize?  We may also want to consider versioning - have you encountered an approach that would allow us to maintain different api versions dynamically? `/api/v1/items` for example